### ISSUE TITLE
Remove header

### ISSUE THREAD

**Issue by danielleogden** [Trillium-Solutions/mendocino-wp-theme#31]
Let's get rid of this almost invisible and unnecessary header. 
![Screen Shot 2021-02-09 at 10 24 01 AM](https://user-images.githubusercontent.com/31776892/107411112-00d41c80-6ac3-11eb-86cc-953003f01fed.png)

**Comment by NomeQ** [comment]
Haha! I didn't even notice it there, but I think is the supposed to be fully-hidden header text, that is there for accessibility. It should just be fully hidden (screen-reader only)

**Comment by NomeQ** [comment]
Good eye, @danielleogden

**Comment by danielleogden** [comment]
@NomeQ oh ok! Glad to hear it has a purpose. I was very confused by it.

**Comment by ekeaton** [comment]
It's gone now! The screen reader text is before that, that text is separate and for tablet/mobile-only and should be hidden on desktop.
